packages/radio/src/react/button.tsx

```diff
@@ -1,21 +1,19 @@
 import { compose, css } from 'glamor'
-import PropTypes from 'prop-types'
-import React, { useState } from 'react'
+import React, { HTMLAttributes, useState, useContext } from 'react'
 
-import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
 import Halo from '@pluralsight/ps-design-system-halo'
-import { useTheme } from '@pluralsight/ps-design-system-theme'
-import { combineFns } from '@pluralsight/ps-design-system-util'
-import { useRadioContext } from './context.js'
-import stylesheet from '../css/index.js'
+import { useTheme, names } from '@pluralsight/ps-design-system-theme'
+import { combineFns, ValueOf } from '@pluralsight/ps-design-system-util'
+import { RadioContext } from './context'
+import stylesheet from '../css'
 
 const styles = {
-  button: ({ disabled }) =>
+  button: (disabled: boolean) =>
     compose(
       css(stylesheet['.psds-radio-button']),
       disabled && css(stylesheet['.psds-radio-button--disabled'])
     ),
-  circle: (themeName, checked) =>
+  circle: (themeName: ValueOf<typeof names>, checked: boolean) =>
```

Comment:
I'd either use Theme.names or alias this themeNames.  Names in this context isn't immediately identifiable.

Reply:
Don't understand what you're saying as as `names` is immediately available, it's imported on line 5????

Reply:
Yes, I see the import.  I'm just saying it's not immediately clear what it is without tracing it to its declaration.  This is an indicator to me that "names" isn't descriptive enough.  "themeNames" or "Theme.names" wouldn't need to be traced to its declaration, because would could reasonably rely on the meaning of the name.